apps/api/src/app/app.module.ts

```diff
@@ -14,7 +14,11 @@ import { HttpExceptionFilter } from './http-exception.filter'
 
 @Module({
   imports: [
-    ConfigModule.forRoot({ isGlobal: true, validate }),
+    ConfigModule.forRoot({
+      isGlobal: true,
+      validate,
+      envFilePath: ['.env', '.env.development']
```

Comment:
How does it choose between .env or .env.development?

Reply:
It's a fallback, if you forget to create the `.env`, the service will still work by picking the `.env.development` file.

It's just to improve developer experience a little bit while we don't have anything sensitive